dbt/models/default/schema/default.vw_pin_sale.yml

```diff
@@ -17,6 +17,25 @@ models:
         description: Indicator for whether or not the observation uses the MyDec sale date
       - name: mydec_deed_type
         description: Deed type from MyDec, more granular than CCAO deed type
+      - name: mydec_line_9_additions
+        description: Binary indicator on the ptax-203 form. A subsection of a question about significant physical changes to the property.
+      - name: mydec_line_9_date_of_significant_change
+        description: The date of the change to the property.
+      - name: mydec_line_9_demolition_damage
+        description: Binary indicator on the ptax-203 form. A subsection of a question about significant physical changes to the property.
+      - name: mydec_line_9_major_remodeling
+        description: Binary indicator on the ptax-203 form. A subsection of a question about significant physical changes to the property.
+      - name: mydec_line_9_new_construction
+        description: Binary indicator on the ptax-203 form. A subsection of a question about significant physical changes to the property.
+      - name: mydec_line_9_no_changes
+        description: >
+          Binary indicator on the ptax-203 form. Indicates "no changes to the property" when coded as 1. A value of 0 may indicate either
+          the presence of changes (subcategories coded as 1) or an unclear status, as many cases have no subcategories coded as 1 despite
+          this being 0.
```

Comment:
@Damonamajor Let me know if this is an adequate description.

Reply:
Looks good to me